lib_collections/models.py

```diff
@@ -217,6 +217,266 @@ class RelatedCollectionPagePlacement(Orderable, models.Model):
     )
 
 
+class ExternalService(models.Model):
+    """
+    Class for link to an external service in a Collection Page.
+    """
+    MENU_OPTIONS = [
+        (1, "LUNA"),
+        (2, "BTAA"),
+    ]
+    service = models.IntegerField(
+        choices=MENU_OPTIONS, default=1, help_text='Choose an external service'
+    )
+    identifier = models.CharField(
+        max_length=255, help_text="identifier to retrieve object from service"
+    )
+
+    panels = [
+        FieldPanel('service'),
+        FieldPanel('identifier'),
+    ]
+
+    class Meta:
+        abstract = True
+
+
+class CollectionPageExternalService(Orderable, ExternalService):
+    """
+    Intermediate class for links to external services in a 
+    Collection Page.  
+    (needed to create an InlinePanel)
+
```

Comment:
Weird line break

Reply:
Is this what you had in mind?

``    """
    Intermediate class for links to external services in a Collection
    Page. (needed to create an InlinePanel)

    """``

Reply:
Is this what you had in mind?

```
    """
    Intermediate class for links to external services in a Collection
    Page. (needed to create an InlinePanel)

    """
```

Reply:
Should be like this:
```
    """
    Intermediate class for links to external services in a Collection
    Page. (needed to create an InlinePanel)
    """
```

Reply:
Oh, good spot!  That extra line break is hard to see!

Reply:
I *think* I caught most of these errant line breaks.  Committing...